django_project/layers/utils.py

```diff
@@ -0,0 +1,31 @@
+from django.db import connection
+
+
+def querying_vector_tile(
```

Comment:
@meomancer can we move this to cloudnative gis? Or use the function from that module.